test/concatenate.test.ts

```diff
@@ -257,24 +257,4 @@ describe('concatenate', () => {
     expect(isShape(a.grad, expectedGradShape)).toBe(true)
     expect(isShape(b.grad, expectedGradShape)).toBe(true)
   })
-  it('gradient from mean', () => {
```

Comment:
why delete this?

Reply:
I actually just moved it into the other test, replacing the other one which used to be just `concatenate` without a `mean`. I realised that that shouldn't be correct, because we should only be calling `backward` on scalars.